client/components/web-preview/style.scss

```diff
@@ -306,3 +307,10 @@ a.web-preview__external.button {
 	margin: 48px 0;
 	height: 100%;
 }
+
+.web-preview__inner .spinner-line {
+	position: absolute;
+		top: 22px;
```

Comment:
Not a fan too to be honest. It's our current guideline: https://wpcalypso.wordpress.com/devdocs/docs/coding-guidelines/css.md#positioning